Rpackage/R/growtree.R

```diff
@@ -0,0 +1,73 @@
+growtree <- structure(function(target.mat, feature.mat, depth=0, maxdepth = Inf,
+                               margin = 0.0, loss="hinge", id = 1L, min_sample = 1,
+                               pred=NULL, side = NULL, weights = NULL) {
+  
+  node_info_print <- NULL
+  ### creating dataframe of info
+  if(!is.null(pred)){
+    node_info_print <- data.frame(pred[[side]], pred[[1]],row.names = "")
+    colnames(node_info_print) <- c("prediction", "cost")
+  }
+  
+  ### passing side = 10 as arguement if node is root.
+  if(side == 10){
```

Comment:
Do you have the guarantee that no other node will have side == 10? For instance, below you have the line `side = 4 + kidid`. Can there be any conflict? If not, why?

Reply:
nope, as pred has 6 list items, kidid is {1,2} as left and right, and side is {5,6}. so we can keep anything other than [1,6]. 

only conflict can be if the splitting is not into left right but into 4-5 different brances, like it happens in classification trees.
But this is regression tree so i guess there shouldnt be any conflict